R/cv_varsel.R

```diff
@@ -190,20 +201,20 @@ cv_varsel.refmodel <- function(object, method = NULL, cv_method = NULL,
       print(paste("Performing the selection using all the data.."))
     }
     sel <- varsel(refmodel,
-      method = method, ndraws = ndraws, nclusters = nclusters,
-      ndraws_pred = ndraws_pred, nclusters_pred = nclusters_pred,
-      cv_search = cv_search, nterms_max = nterms_max - 1,
-      intercept = intercept, penalty = penalty, verbose = verbose,
-      lambda_min_ratio = lambda_min_ratio, nlambda = nlambda, regul = regul,
-      search_terms = search_terms, seed = seed
+                  method = method, ndraws = ndraws, nclusters = nclusters,
+                  ndraws_pred = ndraws_pred, nclusters_pred = nclusters_pred,
+                  cv_search = cv_search, nterms_max = nterms_max - 1,
+                  intercept = intercept, penalty = penalty, verbose = verbose,
+                  lambda_min_ratio = lambda_min_ratio, nlambda = nlambda,
+                  regul = regul, search_terms = search_terms, seed = seed
```

Comment:
Is this indentation an R convention?

Reply:
Not specifically, I think. As you know, setting a specific margin width (80 in projpred, if I saw that correctly) is a general computer science convention. And for the indentation, I follow RStudio's automatic indentation as well as [this style guide](http://adv-r.had.co.nz/Style.html). Commits 4c23b44 to 4ffe4f5 were just performed to make life easier when working in RStudio: I regularly use `Ctrl + A` and `Ctrl + I` to automatically indent the whole code of the current file. Lines which do not follow RStudio's automatic indentation guide yet are then re-indented which causes undesired changes which are not related to the piece of code I am currently working on. So I thought it would be a good idea to re-indent all R files here in this PR so that in the future, re-indenting in RStudio doesn't produce undesired changes which are not related to the piece of code I (or other RStudio users) am/are currently working on.

Reply:
Yes, I enforce 80 characters per line but my indentation system is clearly different, which messes up with my own rules. I'll take a look but I'll most likely accept this one because it seems most people work with RStudio.